in_toto/in_toto_mock.py

```diff
@@ -42,56 +55,23 @@
 log = logging.getLogger("in_toto")
```

Comment:
Is there a reason why you don't use the "a.b.c.." convention when specifying the logger name?
https://docs.python.org/2/library/logging.html#logging.getLogger

Reply:
Yes, there is and it is documented in [`in_toto.log`](https://github.com/in-toto/in-toto/blob/develop/in_toto/log.py#L35-L66) as suggested by the comment.